test/framework/kubernetes.go

```diff
@@ -285,3 +289,73 @@ func checkPodContainerHasEnvVariableWithValue(pod *corev1.Pod, containerName, en
 	}
 	return false
 }
+
+// GetIngressURI returns the ingress URI
+func GetIngressURI(namespace, serviceName string) (string, error) {
+	ingress, err := WaitForIngress(namespace, serviceName)
+	if err != nil || ingress == nil || len(ingress.Spec.Rules) == 0 {
+		return "", fmt.Errorf("Ingress %s does not exist in namespace %s: %v", serviceName, namespace, err)
+	}
+
+	return fmt.Sprintf("http://%s:80", ingress.Spec.Rules[0].Host), nil
+}
+
+// WaitForIngress waits for ingress by a service
+func WaitForIngress(namespace, serviceName string) (*v1beta1.Ingress, error) {
+	ingress := &v1beta1.Ingress{}
+	if exists, err :=
+		kubernetes.ResourceC(kubeClient).FetchWithKey(types.NamespacedName{Name: serviceName, Namespace: namespace}, ingress); err != nil {
+		return nil, err
+	} else if !exists {
+		return nil, nil
+	} else {
+		return ingress, nil
+	}
+}
+
+// ExposeServiceOnKubernetes adds ingress CR to expose a service
+func ExposeServiceOnKubernetes(namespace, serviceName string) error {
+	host := serviceName + namespace
+	if config.IsLocalTests() {
+		host = serviceName
+	}
+
```

Comment:
Please add a possibility to specify hostname using configuration property.
Service name will be added as a hostname prefix.

Reply:
I don't see why adding a new configuration property is needed here as this is for local purposes only as we need to update the "/etc/hosts" file. 
Without the "--local" param, the host will work as expected (appending the namespace into the service name).

Are you suggesting this change because you know that we won't cope with a specific use case running the tests with the "--local" param?

Reply:
After speaking with @sutaakar , his concern is that the "--local" parameter means that we'll be running the tests from local using either a local or remote cluster. Therefore, I'll be adding a new "--local-cluster" to state that the cluster is running locally.